### PR TITLE
doc: add doc for cephfs lazyio

### PR DESCRIPTION
**Pull request by ukernel**
Fixes: http://tracker.ceph.com/issues/38729
Signed-off-by: "Yan, Zheng" <zyan@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/cephfs/lazyio.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+======
+
+LazyIO relaxes POSIX semantics. Buffered reads/writes are allowed even when a
+file is opened by multiple applications on multiple clients. Applications are
+responsible for managing cache coherency themselves.
+
+Libcephfs supports LazyIO since nautilus release.
+
+Enable LazyIO
+=============
```
Comment (leseb) [inline doc/cephfs/lazyio.rst]: Add a newline

File: doc/cephfs/lazyio.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+LazyIO can be enabled by following ways.
+
+- 'client_force_lazyio' option enables LAZY_IO gloablly for libcephfs and
```
Comment (leseb) [inline doc/cephfs/lazyio.rst]:
```suggestion
- ``client_force_lazyio`` option enables LAZY_IO gloablly for libcephfs and
```

File: doc/cephfs/lazyio.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  ceph-fuse mount.
+
+- 'ceph_lazyio(...)' and 'ceph_ll_lazyio(...)' enable LAZY_IO for file handle
```
Comment (leseb) [inline doc/cephfs/lazyio.rst]:
```suggestion
- ``ceph_lazyio(...)`` and ``ceph_ll_lazyio(...)`` enable LAZY_IO for file handle
```

File: doc/cephfs/lazyio.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  in libcephfs.
+
+- 'ioctl(fd, CEPH_IOC_LAZYIO, 1UL)' enables LAZY_IO for file handle in
```
Comment (leseb) [inline doc/cephfs/lazyio.rst]:
```suggestion
- ``ioctl(fd, CEPH_IOC_LAZYIO, 1UL)`` enables LAZY_IO for file handle in
```

File: doc/cephfs/lazyio.rst (hunk 2 above)
Comment (batrick) [reply doc/cephfs/lazyio.rst]: and "gloablly" -> "globally"

**Comment by batrick** [comment]
I changed the base just now.

**Comment by ukernel** [comment]
fixed

**Comment by batrick** [comment]
retest this please

File: doc/cephfs/index.rst [hunk 5]
```diff
@@ -109,6 +109,8 @@ authentication keyring.
     Configuring multiple active MDS daemons <multimds>
     Export over NFS <nfs>
     Application best practices <app-best-practices>
+    Scrub <scrub>
```
Comment (batrick) [inline doc/cephfs/index.rst]: `Scrub...` left in when switching back to master base.

**Comment by batrick** [comment]
ping @ukernel ; please address comments and rebase

**Pull request merged**